kafka-bin-scripts/README.adoc

```diff
@@ -143,23 +116,22 @@ security.protocol=SASL_SSL
 
 sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required \
   username="__<client_id>__" \
-  password="__<client_secret>__" \
-  group.id="__<consumer_group_id>__";
+  password="__<client_secret>__" ;
 ----
 
-NOTE: {product} also supports the SASL/OAUTHBEARER mechanism for authentication, which is the recommended authentication mechanism to use. However, the Kafka bin scripts do not yet fully support OAUTHBEARER, so this example uses SASL/PLAIN.
+NOTE: {product} also supports the SASL/OAUTHBEARER mechanism for authentication, which is the recommended authentication mechanism to use. However, the Kafka scripts do not yet fully support OAUTHBEARER, so this example uses SASL/PLAIN.
 
 --
 . Save the file. You will use it in the next task to connect to your Kafka instance and produce messages.
 
 [id="proc-producing-messages-kafka-bin-scripts_{context}"]
-== Producing messages using Kafka bin scripts
+== Producing messages using Kafka scripts
 
 You can use the `kafka-console-producer` script to produce messages to Kafka topics.
 
 .Prerequisites
 
-* The Kafka bin scripts are downloaded.
+* You have checked that the `kafka-console-producer` and `kafka-topics` scripts are in the `bin/` directory of your Kafka download.
```

Comment:
Not sure we need this. There was something similar in the download step.

Reply:
Why users will have kafka download on the machine? typically they do not run kafka - getting managed version so we might need to rephrase it to actually download kafka binary (telling them where to do it)

Reply:
To answer - once we put proper URL to download we do not need that step at all.

Reply:
Removed